### PR TITLE
[Language] SCSS

### PR DESCRIPTION
**Pull request by MatteuSan**
SCSSで書きました。よろしくお願いします。

#### SCSS:
image

#### ディバグ画面：
image

#### CSS:
image

### CODE REVIEW

**Review by Colk-tech — APPROVED** [review]

File: RADWIMPS.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+@function then() {
+  @return '前';
+}
+
+/// @return {string} 世
+@function 世() {
+  @return '世';
+}
+
+@debug "#{then()then()then()世()}"; // ディバグ画面で書きます。
```
Comment (KisaragiEffective) [inline RADWIMPS.scss]:
```suggestion
@debug "#{then()then()then()世()}"; // デバッグ画面に表示されます。
```

File: RADWIMPS.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+body::after {
+  content: "#{then()}#{then()}#{then()}#{世()}"; // HTMLで書きます。
```
Comment (KisaragiEffective) [inline RADWIMPS.scss]:
```suggestion
  content: "#{then()}#{then()}#{then()}#{世()}"; // HTMLで表示されます。
```

**Review by Colk-tech — CHANGES_REQUESTED** [review]
Plz add a line break at the end!

**Review by KisaragiEffective — APPROVED** [review]
nitpicks about Japanese linguisitics; otherwise LGTM.
Thank you for your contribution!

**Review by Colk-tech — APPROVED** [review]
Thank you for your contribution!

**Comment by KisaragiEffective** [comment]
I'm going to merge this

**Comment by KisaragiEffective** [comment]
Thank you for your contribution!

**Pull request merged**